crates/uv-cli/src/lib.rs

```diff
@@ -122,6 +122,10 @@ pub struct GlobalArgs {
     #[arg(global = true, long)]
     pub toolchain_preference: Option<ToolchainPreference>,
 
+    /// Whether to allow download of uv-managed Python toolchains.
+    #[arg(global = true, long)]
+    pub toolchain_fetch: Option<ToolchainFetchStrategy>,
+
```

Comment:
Should we... hide these? I feel like they're targeting the configuration file more than the CLI

Reply:
Ah I see the interactions are documented on the options. Do those show up in `--help`?

Reply:
Yeah they show up in `--help` for every command since they're global options. The interactions aren't explained actually in `--help`, I guess because they're on a subsequent line. I could add some notes to the CLI flag itself though?

I don't mind having them be available in the CLI but I'm tempted to omit them from the help.

Reply:
Aye yeah that sounds good. I think I'd still prefer to have them in the `--help` personally. Although I see how it might be a bit of a bummer to have them in _every_ help since they're global _and_ because yeah, I agree, these probably won't get used much on the CLI. So I defer to you!